### DISCUSSION TITLE
zed is not in my $PATH

### DISCUSSION THREAD

**Discussion by arifd** [zed-industries/zed#21461] [Feature Requests]
I don't remember how I installed zed, honestly. but it's not in my path.

I would like to run commands like `zed path/to/file`

So I probably need to add `/Applications/Zed.app/Contents/MacOS` to my $PATH

but there is also a bin called `cli` as well as `zed` so I'm wondering what `cli` is for, and if that should be aliased? or something?

**Comment by 5brian** [comment]
you can run `install cli` from the command palette which would let you do `zed path/to/file`

**Reply by arifd** [reply]
Thank you!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
